lib/wallaroo/core/sink/connector_sink/connector_sink.pony

```diff
@@ -305,16 +305,10 @@ actor ConnectorSink is Sink
       // We do not include MessageMsg size overhead in our offset accounting
       _twopc.update_offset(encoded1_len)
 
-      let encoded2 =
-        try
-          let w1: Writer = w1.create()
-          let msg = _notify.make_message(encoded1)?
-          let bs = cp.Frame.encode(msg, w1)
-          Bytes.length_encode(bs)
-        else
-          Fail()
-          encoded1
-        end
+      let w1: Writer = w1.create()
```

Comment:
We're creating a new `Writer` for every message here?  We should hold one in a field and reuse it.

Reply:
It looks like we're doing this elsewhere in the existing code.  Since this isn't a regression, it doesn't need to be addressed in this PR, but if not, can you open an issue?